cfme/tests/infrastructure/test_providers.py

```diff
@@ -136,19 +133,12 @@ def test_host_name_max_character_validation():
         name=fauxfactory.gen_alphanumeric(5),
         hostname=fauxfactory.gen_alphanumeric(256),
         ip_address='10.10.10.13')
-    prov.create()
-    prov.delete(cancel=False)
-
-
-@pytest.mark.tier(3)
-def test_ip_max_character_validation():
-    """Test to validate max character for ip address field"""
-    prov = VMwareProvider(
-        name=fauxfactory.gen_alphanumeric(5),
-        hostname=fauxfactory.gen_alphanumeric(5),
-        ip_address='10.10.10.14')
-    prov.create()
-    prov.delete(cancel=False)
+    try:
```

Comment:
Tis and the test below will probably need some kind of credential added to them.